ci/run_integration_test.sh

```diff
@@ -251,6 +285,25 @@ main() {
 
     run_job
 
+    log INFO "Starting RQD Blender..."
+    docker run -td --name blender \
+    --env CUEBOT_HOSTNAME=cuebot \
+    --volume "/tmp/rqd/shots:/tmp/rqd/shots" \
+    --volume "/tmp/rqd/logs:/tmp/rqd/logs" \
+    -p 8441:8441 \
+    --network opencue_default \
+    opencue/blender
+
+    docker exec blender sh -c 'echo "RQD_USE_IP_AS_HOSTNAME=False" >> /etc/opencue/rqd.conf'
+    log INFO "Restarting RQD Blender..."
+    docker restart blender
+    sleep 3
+
+    add_RQD_tag
+
+    log INFO "Testing Blender job..."
+    run_blender_job
```

Comment:
This PR ends up polluting the integration test script with several blender specific steps. I feel the changes are impactful enough that it makes sense to isolate blender in its own integration test. 

I suggest creating ci/run_blender_integration_test.sh to maintain the general integration test script simple.